packages/protocol/contracts/L1/TaikoToken.sol

```diff
@@ -47,8 +47,8 @@ contract TaikoToken is
         ERC20SnapshotUpgradeable.__ERC20Snapshot_init_unchained();
         ERC20VotesUpgradeable.__ERC20Votes_init_unchained();
 
-        // Mint 1 billion tokens
-        _mint(_recipient, 1_000_000_000 ether);
+        // Mint max Uint128 for testnet.
+        _mint(_recipient, type(uint128).max);
```

Comment:
why max? I think we should stick to 1 billion, it should just work, no?

Reply:
yep it should work, but just thought 1 billion supply with current `10_000 liveness bond` + `20 * 10_000 optimistic proof validity bond`, even if we keep all tokens, we are only able to provide ~5000 blocks optimistic proof under 28 hrs (cooldown window), and if we assume 3 seconds as block time, we may have 1200 blocks per hour, so lets reduce the bond?

Reply:
yep 250 liveness bond makes sense to me, changed: https://github.com/taikoxyz/taiko-mono/pull/15025/commits/2af5f28c3c5aae10be2aa10cb81a5ce5cd9d7396